libres/lib/analysis/ies/ies_enkf.cpp

```diff
@@ -883,14 +729,14 @@ bool ies_enkf_has_var(const void *arg, const char *var_name) {
             return true;
         else if (strcmp(var_name, IES_LOGFILE_KEY) == 0)
             return true;
-        else if (strcmp(var_name, IES_DEBUG_KEY) == 0)
-            return true;
         else if (strcmp(var_name, IES_AAPROJECTION_KEY) == 0)
             return true;
         else if (strcmp(var_name, ENKF_TRUNCATION_KEY) == 0)
             return true;
         else if (strcmp(var_name, ENKF_SUBSPACE_DIMENSION_KEY) == 0)
             return true;
+        else if (strcmp(var_name, IES_DEBUG_KEY) == 0)
```

Comment:
This `else if` seem to be unnecessarily moved. Unless there is a reason to we should probably avoid it to not create a confusing diff.

Reply:
fixed